eng/pipelines/templates/variables/globals.yml

```diff
@@ -8,3 +8,6 @@ variables:
   
   # Disable CodeQL injections except for where we specifically enable it
   Codeql.SkipTaskAutoInjection: true
+
+  # Set the default value so that 'Build Performance Tests' step can resolve it
+  Use.AzcoreFromMain: false
```

Comment:
Looks like this value isn't getting overwritten when I define it queueing up a pipeline run.

Reply:
It's possible my suggested fix isn't quite working. Do you have a link to the pipeline run? It doesn't seem to get linked to the PR (azure pipelines has been dropping the ball here lately).

Reply:
[Here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3046116&view=results) is the run from last night.

I'm looking at [these docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#runtime-expression-syntax) and it seems like the `$[variables.foo]` syntax is what we want.  I tried that and it gets us closer but still not completely there.  Latest run [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3048169&view=logs&j=730d91e9-ccca-5f92-9343-3b1124b19442&t=57014ab0-c321-5206-21d1-de9826f53695).

Reply:
Looks like defining a var in the YAML [won't allow it to be overridden](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#allow-at-queue-time) at queue time.